### ISSUE TITLE
landing stall

### ISSUE THREAD

**Issue by FlandreScarlet1** [giuliodondi/kOS-ShuttleEntrySim#10]
for some reason when i try landing with the script the shuttle flips

**Comment by giuliodondi** [comment]
First of all, stall is not the correct word  to describe your situation.
"Stall" means that the wing does not generate lift anymore. "Flip" means that the lift indeed produced by the wing generates a positive pitching moment about the c.g. (i.e. nose up).

If you want to know exactly the reason your "flip" happens, [you can find it in section 4.1.1.3 of the Shuttle aerodynamic data book](https://archive.org/details/nasa_techdoc_19810067693). In plain english, the Shuttle is pitch-unstable when subsonic. 

The script does try to keep it stable but, since you provided no information surrounding your incident, I have no idea if the script could have done it in your situation.
You need to balance your OMS fuel and payload right. You need to anticipate this tendency and use flap trim, or use the automatic feature of the script. 
You also need to be careful with your control inputs, especially if you use a keyboard since you can only give full inputs with key presses
